templates/tanstack-shell/app/components/LoginForm.tsx

```
@@ -0,0 +1,161 @@
+              <p className="form-error">{field.state.meta.errors[0]}</p>
+            )}
+          </div>
+        )}
+      />
+
+      <form.Field
+        name="password"
+        validators={{
+          onChange: zodValidator(loginSchema.shape.password),
+        }}
+        children={(field) => (
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)